### PR TITLE
Deere 2.1 layout & color fixes

### PR DESCRIPTION
**Pull request by ronso0**
- removing now obsolete FullControlsRow
- equalize padding/layout-spacing in contrlos row & overview row

I closed #14 and included the commit here

### CODE REVIEW

**Comment by nopeppermint** [comment]
it seems to me like there are some more color fixes required (if the goal is to have everything selected with a blue background):
before this fix:
linux_mint_18 2
with this fix:
dere_new_fix

deree_fix_3


So basically:
- the preview button 
- some empty spaces on the left (AutoDJ/Crates, Playlists, Crates/New Crates, Computer/all submenus, Recordings, History/all submenus, Analyse)

**Comment by ronso0** [comment]
> So basically:
>   - the preview button
>   - some empty spaces on the left (AutoDJ/Crates, Playlists, Crates/New Crates, ...)

Ooh, will fix the preview button.
But I don't know what to do about that empty space in Lib tree. I can recall that I tried to fix that in Tango ages ago

**Comment by ronso0** [comment]
For the branches in Lib tree I could fix it, but for the Preview button there's already a style hack in legacyskinparser.cpp. Right now I'm trying if this is avoiding qss style.

**Comment by ronso0** [comment]
I fixed the empty space in tree.
But I don't find a way to define the Preview button background according to row selection.
For now I used the 'alternate-background-color' (every second line in Library) an change it to blue if the respective track is playing. This looks nice and may even help to find the playing track in case the Preview deck is not visible and that track is scrolled out of view.

Maybe someone else has an idea how to fix this with qss only.

**Comment by nopeppermint** [comment]
thanks @ronso0 now it looks like this:
with_fix

File: res/skins/Deere/style.qss [hunk 1]
```diff
@@ -212,16 +215,17 @@ QTableView::indicator:unchecked {
...
 }
 
 QPushButton#LibraryPreviewButton:!checked {
   image: url(skin:/image/style_library_preview_play.png);
+  background-color: #1A1A1A;
 }
 
 QPushButton#LibraryPreviewButton:checked {
   image: url(skin:/image/style_library_preview_pause.png);
+  background-color: #006596;
```
Comment (nopeppermint) [inline res/skins/Deere/style.qss:228]: I suggest 

"228 background-color: #006596;"

then It looks like this:
new_fix

which is (in my opinion) better than

with_fix

Comment (ronso0) [reply res/skins/Deere/style.qss:228]: Blue might be better if Preview deck is not visible and if you scroll around a lot while previewing.
Dark grey looks better but breaks the highlighted line, depending on how columns are sorted.
I don't mind..

Comment (ronso0) [reply res/skins/Deere/style.qss:228]: @Be-ing What do you think?

**Comment by ronso0** [comment]
Don't merge yet, I'll add commits to fix the flickering when toggling parallel waveforms I discovered while checking [Bug #1731791](https://bugs.launchpad.net/mixxx/+bug/1731791)

**Comment by ronso0** [comment]
Done.

**Comment by ronso0** [comment]
= Ready to merge

**Comment by ronso0** [comment]
@Be-ing Did you have a look at this yet?
Let's decide about the Preview button,

then merge this so Deere update get's finished

File: res/skins/Deere/style.qss (hunk 1 above)
Comment (Be-ing) [reply res/skins/Deere/style.qss:228]: I think the background of the preview icon for a deck playing in the preview deck should be different from both the highlight background color and the unselected background color.

Comment (Be-ing) [reply res/skins/Deere/style.qss:228]: Actually, I like it how it is. It jumps out a little bit, but not too much.

**Comment by Be-ing** [comment]
LGTM thank you

**Pull request merged**